include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1495,8 +1495,18 @@ FEPointEvaluation<n_components, dim, spacedim, Number>::do_reinit()
                                               current_face_number);
 
   const_cast<unsigned int &>(n_q_points) =
-    n_q_points_scalar / n_lanes_user_interface +
-    (n_q_points_scalar % n_lanes_user_interface > 0 ? 1 : 0);
+    (n_q_points_scalar + n_lanes_user_interface - 1) / n_lanes_user_interface;
+
+  if (update_flags & update_values)
+    values.resize(n_q_points, numbers::signaling_nan<value_type>());
```

Comment:
`resize()` and not `assign` is wanted? It feels odd that only the new entries are set to nan.